### DISCUSSION TITLE
Can't hold down left mouse button when hovering with a tablet

### DISCUSSION THREAD

**Discussion by nuewink** [ppy/osu#27310] [Q&A]
### Type

Game behaviour

### Bug description

When I tried to do the tablet + mouse playstyle in standard (clicking on the mouse and aiming with a tablet), I noticed that I can't click some sliders, the slider ticks would always miss. I looked at the key counter display and noticed that holding down left mouse button will only click it once. The moment you click left mouse button, it will automatically release. Once you put the pen away from the tablet surface, left clicks work normally again. Right clicks are unaffected.
The issue persists in the main menu of lazer, which you can see if you look at the cursor and if you listen closely to the click sound.
The issue does NOT appear if you disable "Tablet" in the osu!lazer settings.
The issue does NOT appear in osu!stable.

### Screenshots or videos

https://www.youtube.com/watch?v=pAONLpWbgT4

### Version

2024.219.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14362062/compressed-logs.zip)

**Comment by nuewink** [comment]
Update: still acts that way on 2024.221.0

**Comment by frenzibyte** [comment]
I can not reproduce this on my CTL-472 on both macOS and Windows via VM. Please specify the tablet you're using here. I would also suggest turning off anything interacting with the tablet outside of the game (i.e. the `OpenTabletDriver` you have running outside or anything like that)

**Reply by nuewink** [reply]
I am using a CTL-472 on Windows 10. After disabling OpenTabletDriver, the issue persists.

**Reply by RadekMojzis** [reply]
I am having the same issue.
When the tablet mode is enabled in osu, if the cursor is moved by the tablet, the left mouse button is immediately released.
My tablet is "one by wacom S" it has been this way ever since the tablet option was introduced and it happens to me on both windows 10 and 11 on both my desktop and laptop.
